.github/workflows/rust.yml

```diff
@@ -473,13 +473,7 @@ jobs:
         with:
           path: /github/home/.cargo
           # this key equals the ones on `linux-build-lib` for re-use
-          key: cargo-cache-
-      - name: Cache Rust dependencies
```

Comment:
As this job runs `cargo fmt` there is no reason to restore the contents of `target` as part of the job